owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListViewModel.kt

```diff
@@ -67,9 +67,10 @@ class MainFileListViewModel(
     private val getFileByRemotePathUseCase: GetFileByRemotePathUseCase,
     private val getSpaceWithSpecialsByIdForAccountUseCase: GetSpaceWithSpecialsByIdForAccountUseCase,
     private val sortFilesWithSyncInfoUseCase: SortFilesWithSyncInfoUseCase,
+    private val synchronizeFolderUseCase: SynchronizeFolderUseCase,
+    private val getSpacesFromEveryAccountUseCase: GetSpacesFromEveryAccountUseCase,
```

Comment:
vThere's no specific use case for that. There's a query in `SpacesDao`, but no use case associated to it. IMO, at this point the best is to keep this as technical debt because in the short term this new issue will be addressed: https://github.com/owncloud/android/issues/3971, making this being deleted.